GenderPayGap.WebUI/Views/AddOrganisationManualChooseSector/ManualChooseSector.cshtml

```diff
@@ -0,0 +1,147 @@
+﻿@using GenderPayGap.WebUI.Models.AddOrganisation
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.AddOrganisation.AddOrganisationManualViewModel
+@{
+    ViewBag.Title = "What type of organisation you would like to add? - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        Model.Validate = null; // Required to make the "Back" link work correctly
+    }
+    @(Html.GovUkBackLink(new BackLinkViewModel
+    {
+        Text = "Back",
+        Href = Url.Action("ManualReview", "AddOrganisationManualReview", Model),
+        
+        // Note: some of the "Back" links in this journey can't use the default browser behaviour
+        //  so we should control all of them explicitly
+        OverrideWithJavascript = false
+    }))
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <form method="get"
+              action="@Url.Action("ManualChooseSector", "AddOrganisationManualChooseSector")">
+
+            @(Html.GovUkErrorSummary())
+
+            @(Html.HiddenFor(m => m.Query))
+            @(Html.HiddenFor(m => m.OrganisationName))
+            @(Html.HiddenFor(m => m.PoBox))
+            @(Html.HiddenFor(m => m.Address1))
+            @(Html.HiddenFor(m => m.Address2))
+            @(Html.HiddenFor(m => m.Address3))
+            @(Html.HiddenFor(m => m.TownCity))
+            @(Html.HiddenFor(m => m.County))
+            @(Html.HiddenFor(m => m.Country))
+            @(Html.HiddenFor(m => m.PostCode))
+            @(Html.HiddenFor(m => m.IsUkAddress))
+            @if (Model.SicCodes != null)
+            {
+                @for (int i = 0; i < Model.SicCodes.Count; i++)
+                {
+                    <input type="hidden" name="@(nameof(Model.SicCodes))" value="@(Model.SicCodes[i])" />
+                }

```

Comment:
Might be a bit cleaner to use a foreach with something like 

```
foreach( var sicCode in Model.SicCodes.Select(x,i) => new {Code = x, Index = i){
    <input type="hidden" name="@nameof(sicCode.Code)" value="@(sicCode.Index)" />
}
```

Again this is just personal preference, so I'm not sure whether my suggestion is worth following!

Reply:
I'm not sure why I thought these needed to be for loops.

I've changed these to be:
```cshtml
@foreach (int sicCode in Model.SicCodes)
{
    <input type="hidden" name="@(nameof(Model.SicCodes))" value="@(sicCode)" />
}
```